dGame/dUtilities/VanityUtilities.cpp

```diff
@@ -22,9 +22,18 @@
 
 #include <fstream>
 
-std::vector<VanityObject> VanityUtilities::m_Objects = {};
-std::set<std::string> VanityUtilities::m_LoadedFiles = {};
 
+namespace {
+	std::vector<VanityObject> m_Objects;
```

Comment:
Nit: Should we be using the member naming convention outside a class?

Reply:
no clue, as below, I was instructed on these changes by emo

Reply:
Member naming should be removed here to jades point, they aren't members anymore

Reply:
But if u want to jeep the cl small, feel free to push it back

Reply:
ah missed what this mean, changed